README.md

```diff
@@ -1199,16 +31,16 @@ public function main() {
 
 2. Download and install [Ballerina](https://ballerina.io/). 
 
-### Building the Source
+### Building the source
 
-Execute the commands below to build from the source after installing Ballerina SLP8 version.
+Execute the commands below to build from the source after installing the relevant Ballerina version. ( Run the following commands after setting the `eventhub` directory as the working directory )
```

Comment:
Since installing Ballerina is a prerequisite, we do not need to mention it here right?

Reply:
Run the following commands after setting the `eventhub` directory as the working directory -> setting the `eventhub` directory  should be a step, and mention how to do it.

Reply:
This line is removed and replaced with the sentence in the common format. 
https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit?usp=sharing